rfc/rfc011-verifiable-credential.md

```diff
@@ -0,0 +1,199 @@
+This also means that specifications SHOULD to be updated when new proof types are available.
+
+### 4.4 Trust
+
+It MUST list the requirements for when a VC is to be trusted. This can be any combination of configuration, other VCs and/or automatic mechanism.
+
+### 4.5 Revocation
+
+It MUST specify how a VC can be revoked by the issuer, or it MUST specify an expiration duration. It MAY refer to the default revocation mechanism stated below.
+
+### 4.5.1 Default revocation
+
+VCs that are issued by a Nuts DID can be revoked by publishing the following transaction on the network:
+
+```json
+{
+  "issuer": "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY",
+  "subject": "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467",
+  "currentStatus": "Revoked",
+  "statusReason": "Disciplinary action",
+  "statusDate": "2010-02-01T19:73:24Z",
+  "proof": {
+    "type": "JsonWebSignature2020",
+    "created": "2017-06-18T21:19:10Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467#qjHYrzaJjpEstmDATng4-cGmR4t-_V3ipbDVYZrVe4A",
+    "jws": "eyJhbGciOiJSUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..TCYt5X
+    sITJX1CxPCT8yAV-TVkIEq_PbChOMqsLfRoPsnsgw5WEuts01mq-pQy7UJiN5mgRxD-WUc
+    X16dUEMGlv50aqzpqh4Qktb3rk-BuQy72IFLOqV0G_zS245-kronKb78cPN25DGlcTwLtj
+    PAYuNzVBAh4vGHSrQyHUdBBPM"
+  }
+}
+```
+
+Such a revocation transaction has the following requirements:
+
+* the **issuer** MUST match the DID as the `issuer` field of the VC.
+* the **subject** MUST match the `id` field of the VC.
+* the **currentStatus** MUST be changed to `Revoked`.
+* the **statusReason** MAY be filled with a revocation reason.
+* the **statusDate** MUST provide the date in rfc3339 format. From this moment in time the VC is revoked.
+* the **proof** MUST be a `JsonWebSignature2020` proof.
+
+The transaction MUST be published on the Nuts network. It MAY use the `verificationMethod` for the transaction.
+The content-type is `application/vc+json;type=revocation`
+
+### 4.6 Use cases
+
+It MUST specify where the VC SHOULD be used: as requirement for other VCs, in the oauth flow according to [RFC003](rfc003-oauth2-authorization.md) or any other use case.
+If a VC can be used in a Verifiable Presentation (VP), it MUST specify if additional VCs are to be expected in the VP.
+It SHOULD describe the use case well enough so any implementation can take appropriate measures, such as indexing certain fields.
+
+### 4.7 Privacy considerations
+
+It MUST specify if a VC SHOULD be published via some means. It MUST specify if it contains personal information as identified by the GDPR or local legislation. 
+It MUST specify who MAY receive and/or store the VC.
+
+### 4.8 Services
+
+It MUST specify if other services than the Nuts network are required for the VC to be issued and/or obtained. If so, it MUST specify the protocol.
```

Comment:
Isn't this part of the DID method? So it should specify a list of DID methods e.g. "nuts", "web", "peer", "sov"?